### ISSUE TITLE
PSDD Model Library fails to validate

### ISSUE THREAD

**Issue by j26151** [ZeligsoftDev/CX4CBDDS#338]
# **Issue and tracking information**

### Developer's Actual time spent on fix (hours)  

# **Issue reporter to provide a detailed description of the issue in the space below**

Our example using the PSDD model library does not validate. 
![image](https://user-images.githubusercontent.com/49879646/140787925-636388b1-eb4b-4197-a83f-e54e88e5cb0d.png)
I believe the problem is this line here: https://github.com/ZeligsoftDev/CX4CBDDS/blame/3b99c87145f1a561c0367b216fec0707346616b2/bundles/com.zeligsoft.domain.dds4ccm.psddconnector/models/CCM_PSDD.uml#L73  and this query is null.

Looking into the middleware for AXCIOMA, I don't see any attribute for query:
https://github.com/RemedyIT/ciaox11/blob/master/connectors/psdd4ccm/idl/ccm_psdd.idl#L33

I am not sure how this didn't break before but It is throwing errors now

**Comment by ysroh** [comment]
Is this gating for the release? Let me know so I can work on it. Thanks.

**Comment by J17359** [comment]
> Is this gating for the release? Let me know so I can work on it. Thanks.

Shooting an email now to you all to summarize our vision for what issues make it into the release. I figured that would be easier than commenting on the individual tickets.

**Comment by ysroh** [comment]
Sounds good, Thanks!

**Comment by ysroh** [comment]
Here is the updated PSDD library. Can you please take a look to see if it is o
[PSDD.zip](https://github.com/ZeligsoftDev/CX4CBDDS/files/7499574/PSDD.zip)
k?

**Comment by j26151** [comment]
I imported your models into my workspace and overrode the installed pathmap for PSDD with the contents of your zip file. After that, I repaired module instantiations for our PSDD example and everything validated and generated correctly. I was also able to build using the generated idl and taginfo.xml files. I believe the updated PSDD library here fixes the issue we were seeing, although I will want to test again after it has been merged into the codebase and automatically installed into papyrus.
